systemd/ocp-cluster-ca.sh

```diff
@@ -15,11 +15,8 @@ custom_ca_path=/opt/crc/custom-ca.crt
 external_ip_path=/opt/crc/eip
 
 if [ ! -f ${custom_ca_path} ]; then
-    echo "Cert bundle /opt/crc/custom-ca.crt not found, generating one..."
-    # generate a ca bundle and use it, overwrite custom_ca_path
-    CA_SUBJ="/OU=openshift/CN=admin-kubeconfig-signer-custom"
-    openssl genrsa -out /tmp/custom-ca.key 4096
-    openssl req -x509 -new -nodes -key /tmp/custom-ca.key -sha256 -days 365 -out "${custom_ca_path}" -subj "${CA_SUBJ}"
```

Comment:
does that mean we don't need to generated the custom ca bundle key and it is user's who should take care of that? I am not sure how  this is part of this commit, may be a separate one?

Reply:
Yeah, when using the bundle with `mapt` it is created by `mapt` with crc or directly using with vfkit,libvirt users have to create this.. we can automate it when using with `crc`